packages/scandipwa/src/store/Cart/Cart.dispatcher.js

```diff
@@ -30,6 +30,8 @@ export class CartDispatcher {
     async updateInitialCartData(dispatch, isForCustomer = false) {
         // Need to get current cart from BE, update cart
         try {
+            dispatch(updateIsLoadingCart(true));
+
             // ! Get quote token first (local or from the backend) just to make sure it exists
 
             dispatch(updateIsLoadingCart(true));
```

Comment:
You are making the same action twice. Why?

Reply:
I merged the master branch and haven't uploaded the edit after merging yet - still adding more features